### ISSUE TITLE
Update Activity Feeds

### ISSUE THREAD

**Issue by timolegros** [hicommonwealth/commonwealth#7691]
## Description
The activity feeds ('For You', 'Global', 'ChainEvents') are all based on the `Notifications` table which will soon be removed as part of our transition Knock. As such, we need to modify how we fetch the data to display the feeds.

We should modify the query so that we display the 10 most recently created threads and the 10 most recently created comments. For the global feed this can be any comment and for the 'For You' feed it should be limited to new threads and comments from communities the user has joined. This almost replicates the original behaviour except that before we retrieved the top most recently created threads/comments rather than the top 10 of each (meaning in the old fetch you could technically end up with only threads if no comments were recently created).

I also recommend that we remove the chain-event feed. It shows exactly the same thing as the notifications but rebuilding the feed would take a considerable amount of work (probably 5 pointer) since chain-events are no longer stored in the DB and neither will chain-event notifications (would need to either use a new Knock workflow or created a projection in the DB).

**Comment by timolegros** [comment]
Scooting this up in priority because https://github.com/hicommonwealth/commonwealth/issues/8006 will be blocked by the notifications table since it contains full thread data and would need to be refactored if we moved thread content to external storage.

**Comment by timolegros** [comment]
Ahhhhh another ticket that went way down in points. Malik paved the way on this one with #7958.
